tap_tiktok_ads/streams.py

```diff
@@ -145,7 +145,7 @@ def transform_ad_management_records(records, bookmark_value):
         # In case of an adgroup request, transform 'is_comment_disabled' type from integer to boolean
         if 'is_comment_disable' in record:
             record['is_comment_disable'] = bool(record['is_comment_disable'] == 0)
-        if bookmark_value is None or record['modify_time'] > bookmark_value:
+        if bookmark_value is None or utils.strptime_to_utc(record['modify_time']) > utils.strptime_to_utc(bookmark_value):
```

Comment:
Add PR comment here regarding the need for the above date transform.

Reply:
added comment